Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_botiboti/src/main.rs

```diff
@@ -0,0 +1,19 @@
+use std::io;
+
+fn calc(age: i32) -> i32 {
+    age * 365
```

Comment:
This can overflow.
```
❯ cargo run
Enter age.
1000000000
You entered: 1000000000.
thread 'main' panicked at src/main.rs:4:5:
attempt to multiply with overflow
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```
We have to make sure the types support al possible operations on them. Options:
- validate input
- make the result type bigger
- return a Maybe or Result
- use bigint
I lean towards the last one.